src/utils/parser.ts

```diff
@@ -39,3 +40,41 @@ export function nodeToString(node: Node, code: string) {
   // Copy the string value directly from the input code
   return code.slice(start, end).trim();
 }
+
+/**
+ * Finds all css delcarations
+ * @param root PostCss AST Root
+ */
+function getCssProps(root: Root): Declaration[] {
+  const { nodes } = root;
+
+  if (!nodes) {
+    return [];
+  }
+
+  function iterate(nodes: ChildNode[], declarations: Declaration[] = []) {
+    nodes.forEach((item) => {
+      if (item.type === "decl") {
+        declarations.push(item);
+      } else if (item.type === "rule" && item?.nodes) {
+        iterate(item.nodes, declarations);
+      }
+    });
+
+    return declarations;
+  }
+
+  return iterate(nodes);
+}
+/**
+ * Parses css into PostCss AST and returns array of declarations
+ * @param css css string
+ */
+export function parseCss(css: string): Declaration[] {
+  try {
+    const root: Root = parse(css);
+    return getCssProps(root);
+  } catch (e) {
+    throw new Error("Could not parse CSS");
```

Comment:
I'm curious what kind of error messages are thrown by postcss, could they be useful to the user?

Reply:
Good call. I have updated

![image](https://user-images.githubusercontent.com/15922558/82624271-7c382a00-9c25-11ea-9e6d-fff8512209c7.png)